node/middlewares/errorHandler.ts

```diff
@@ -0,0 +1,18 @@
+export async function errorHandler(ctx: Context, next: () => Promise<void>) {
```

Comment:
Since there is no error to be thrown in the subsequent middleware, I think we can remove this. Or we can check there if `countryCode` is empty, than throw an error and keep this.